index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1><b>Leaderboard</b></h1>
+    <div class="content">
+      <section class="score-head">
+        <div class="refresh-section">
+          <h2>Recent scores</h2>
+          <button type="button" id="refreshBtn" class="btn black-border">Refresh</button>
+        </div>
+        <ul class="scores" id="scores">
+
+        </ul>
+      </section>
+      <section class="add-score">
+        <h2>Add your score</h2>
+        <form>
+          <input type="text" name="name" id="name" class="black-border form-input">
+          <input type="number" name="score" id="score" class ="black-border form-input">
+          <input type="submit" value="Submit" id="form-submit" class="btn black-border submit-btn">
```

Comment:
- [ ] When I submit the new score, the score uploads to the API, but this doesn't auto-refresh the score list. I need to refresh the page or something like that to see the new Score, let's make this better by adding an auto-refresher when you submit a new score. 💪🏾 This will also prevent double entries 😄

Reply:
there  is a refresh button for that

Reply:
but I'll try to implement that

Reply:
I tried and it's like the delay for the API call is longer than the time the submitBtn takes to execute the function

Reply:
Ok thanks very much for your review i fixed it